rsconnect/api.py

```diff
@@ -354,8 +354,8 @@ def app_access(self, app_guid: str) -> None:
         if self.is_app_failed_response(response):
             raise RSConnectException(
                 "Could not access the deployed content. "
-                + "The app might not have started successfully. "
-                + "Visit it in Connect to view the logs."
+                + "The app might not have started successfully."
+                + f"\n\t For more information: {self.app_config(app_guid)['logs_url']}"
```

Comment:
The paranoid programmer in me says to avoid the possible `KeyError` before the exception is raised.

```suggestion
                + f"\n\tFor more information: {self.app_config(app_guid).get('logs_url')}"
```

Reply:
(1) yes, agreed about the key error (2) the ` ` there is (sadly) intentional to avoid needing to `s/\t /\t/g` elsewhere